pkg/apis/pipeline/v1beta1/pipeline_types.go

```diff
@@ -252,6 +259,13 @@ type PipelineTask struct {
 	// Note: PipelineSpec is in preview mode and not yet supported
 	// +optional
 	PipelineSpec *PipelineSpec `json:"pipelineSpec,omitempty"`
+
+	// OnError defines the exiting behavior of a PipelineRun on error
+	// can be set to [ continue | stopAndFail ]
+	// Note: OnError is in preview mode and not yet supported
+	// TODO(#7165)
+	// +optional
+	OnError PipelineTaskOnErrorType `json:"onError,omitempty"`
```

Comment:
are we still adding features to v1beta1 types or just v1?

Reply:
Great point. My understanding that it is sufficient to only add them in v1 since v1beta1 has been marked as deprecated.

Reply:
Hi @dibyom and @JeromeJu , was wondering if this addresses your question and if we can merge this PR 😁 ?

Reply:
Thanks Quan. In this case that SGTM